### PR TITLE
OF-1020: Admin Console Remote File Inclusion

### PR DESCRIPTION
**Pull request by richmidwinter**

### CODE REVIEW

File: src/java/org/jivesoftware/openfire/update/UpdateManager.java [hunk 1]
```diff
@@ -73,7 +73,7 @@
     /**
      * URL of the servlet (JSP) that provides the "check for update" service.
      */
-    private static String updateServiceURL = "http://www.igniterealtime.org/projects/openfire/versions.jsp";
+    private static String updateServiceURL = "https://www.igniterealtime.org/projects/openfire/versions.jsp";
```
Comment (akrherz) [inline src/java/org/jivesoftware/openfire/update/UpdateManager.java]: The only concern here would be that the JRE supports LetsEncrypt signed certs that igniterealtime uses.

Comment (dwd) [reply src/java/org/jivesoftware/openfire/update/UpdateManager.java]: True. We could (possibly) use a specific truststore for this that explicitly (and maybe only) included the LetsEncrypt CA.

Comment (guusdk) [reply src/java/org/jivesoftware/openfire/update/UpdateManager.java]: Oh, let's not go down the road of maintaining more certs, keeping it in sync with whatever we use on the website.

What exactly are we trying to achieve here? Is doing HTTPS with certificate-checks disabled acceptable?

**Comment by guusdk** [comment]
This looks like a sensible fix. Thanks!

**Pull request merged**